### PR TITLE
Added notes for changes related to k8s 1.22 in rancher-backup app

### LINKED ISSUE #3694: [Rancher2] Add notes for changes related to  k8s 1.22  in rancher-backup app
**Request Summary:**

K8s 1.22 drop the apiVersion `apiextensions.k8s.io/v1beta1`. The rancher-backup app is updated to support k8s 1.22, and there are some breaking changes related to the process of making or restoring backups. 

**Details:**

- Backup files that contain CRDs which apiVesion is `apiextensions.k8s.io/v1beta1` can not be restored into a 1.22 cluster properly because 1.22 does not support the apiVersion anymore. 
- The default `resouceSet` in the rancher-backup app is updated to collect only CRDs that uses `apiextensions.k8s.io/v1` 

We also need to add the following general rules for rancher-backup app somewhere in the docs:
- Versions:
	- the 2.x.x is for Rancher v2.6.x line
	- the 1.x.x is for Rancher v2.5.x line
- Restoring should happen between the same Rancher version. For example, if the backup file is made from a Rancher 2.6.2 setup, we can use the backup to either restore the same setup or migrate Rancher 2.6.2 to a new cluster 
- The K8s version should also be considered when restoring a backup since the supported apiVersion in the cluster and in the backup file could be different (we just hit this problem in 1.22)

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3694

References: 
https://rancher.com/docs/rancher/v2.6/en/backups/back-up-rancher/
https://rancher.com/docs/rancher/v2.6/en/backups
https://rancher.com/docs/rancher/v2.6/en/backups/migrating-rancher/

Also: https://rancher.com/docs/rancher/v2.5/en/backups/back-up-rancher/ (added versioning note)

### CODE REVIEW

File: content/rancher/v2.5/en/backups/back-up-rancher/_index.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ In this section, you'll learn how to back up Rancher running on any Kubernetes c
 
 The backup-restore operator needs to be installed in the local cluster, and only backs up the Rancher app. The backup and restore operations are performed only in the local Kubernetes cluster.
 
-> When restoring a backup into a new Rancher setup, the version of the new setup should be the same as the one where the backup is made.
+Note that the Rancher app version 1.x.x is for the Rancher v2.5.x line.
```
Comment (btat) [inline content/rancher/v2.5/en/backups/back-up-rancher/_index.md:14]:
```suggestion
Note that rancher-backup operator version 1.x.x is for the Rancher v2.5.x.
```

File: content/rancher/v2.6/en/backups/back-up-rancher/_index.md [hunk 2]
```diff
@@ -5,12 +5,18 @@ weight: 1
 
 In this section, you'll learn how to back up Rancher running on any Kubernetes cluster. To backup Rancher installed with Docker, refer the instructions for [single node backups]({{<baseurl>}}/rancher/v2.6/en/backups/docker-installs/docker-backups)
 
-The backup-restore operator needs to be installed in the local cluster, and only backs up the Rancher app. The backup and restore operations are performed only in the local Kubernetes cluster.
+The backup-restore operator needs to be installed in the local cluster, and only backs up the Rancher app. The backup and restore operations are performed only in the local Kubernetes cluster. 
+
+Note that the Rancher app version 2.x.x is for the Rancher v2.6.x line.
```
Comment (btat) [inline content/rancher/v2.6/en/backups/back-up-rancher/_index.md:10]:
```suggestion
Note that rancher-backup operator version 2.x.x is for the Rancher v2.6.x line.
```

**Review by btat — CHANGES_REQUESTED** [review]

File: content/rancher/v2.5/en/backups/back-up-rancher/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.5/en/backups/back-up-rancher/_index.md]: Revised as suggested (took out "the").

File: content/rancher/v2.6/en/backups/back-up-rancher/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/backups/back-up-rancher/_index.md]: Revised as suggested (took out "line").

**Review by btat — APPROVED** [review]

**Pull request merged**